_layouts/repo_instance.html

```diff
@@ -18,15 +18,16 @@
     <div class="row">
       &nbsp;
     </div>
-    <div class="row">
-      {% include repo_info.html %}
-    </div>
+
   </div>
 </div>
 
 {% for distro in page.all_distros %}
   <div class="distro distro-{{distro}}">
     <div class="container-fluid">
+      <div class="row">
+        {% include repo_info.html %}
```

Comment:
class "row" adds negative margins, which pushes this div to be unaligned vertically with the rest of the content. I believe it works fine if you simply remove the "row" class here. You can actually see this in the demo picture that you showed.